Branch-SDK/src/io/branch/referral/util/BranchViewHandler.java

```diff
@@ -311,7 +273,7 @@ public void updateUsageCount(Context context, String branchViewID) {
         /**
          * Called when user click the positive button on Branch view
          *
-         * @param action action name associated with the App Branch item
+         * @param action       action name associated with the App Branch item
```

Comment:
Internally BranchViewObject is associated  with its on action (or Event name). It is added to provide the action  name in the callback.